graylog2-web-interface/src/components/users/PaginatedItem.jsx

```diff
@@ -20,13 +24,33 @@ const Container: StyledComponent<{}, ThemeInterface, HTMLSpanElement> = styled.s
 
 const Header = styled.h4`
   padding-bottom: 5px;
+  flex: 1;
 `;
 
-const PaginatedItem = ({ item: { name, description } }: Props) => (
-  <Container>
-    <Header>{name}</Header>
-    <span>{description}</span>
-  </Container>
-);
+const Description = styled.span`
+  flex: 2;
+`;
+
+const StyledDeleteButton = styled(IconButton)(() => css`
+  flex: 0;
+`);
```

Comment:
Don't need the inner function if there's no props or theme being used

```suggestion
const StyledDeleteButton = styled(IconButton)`
  flex: 0;
`;
```